src/components/auction/OrderPlacement/index.tsx

```diff
@@ -326,18 +341,33 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
     !!account &&
     !!biddingToken.address
 
+  const auctioningTokenAddress = auctioningToken && auctioningToken?.address
+  const linkForKYC = auctioningTokenAddress ? kycLinks[auctioningTokenAddress] : null
   return (
     <>
       <Wrapper>
         {auctionInfoLoading && <InlineLoading size={SpinnerSize.small} />}
-        {!auctionInfoLoading && isPrivate && !signatureAvailable && (
+        {!auctionInfoLoading && isPrivate && !signatureAvailable && account == null && (
+          <ActionButton onClick={toggleWalletModal}>Connect Wallet</ActionButton>
+        )}
+        {!auctionInfoLoading && isPrivate && !signatureAvailable && account != null && (
           <PrivateWrapper>
             <LockBig />
             <TextBig>Private auction</TextBig>
             <EmptyContentTextNoMargin>
-              You are not allowed to place an order.
+              You need to get allowed to participate.
             </EmptyContentTextNoMargin>
-            <EmptyContentTextSmall>Ask the auctioneer to get allow-listed.</EmptyContentTextSmall>
+            <EmptyContentTextSmall>
+              {linkForKYC ? (
+                <ExternalLink href={linkForKYC}>
+                  <h2>Get Allowed ↗</h2>
```

Comment:
Maybe the designers don't like to have a `h2` here 😂, we could get the h2 style at `LinkCSS` or maybe use the same style of `Connect Wallet button`

Reply:
Yes, we should modify it. Though my css styles are very bad.
@maria-vslvn could you find a proper solution that looks somehow similar to the h2?